doc/user/deploy.rst

```diff
@@ -9,66 +9,52 @@ It's a great way to quickly test new libraries without cluttering your
 global site-packages or run multiple projects on the same machine which 
 depend on a particular library but not the same version of the library.
 
-Install virtualenv
+Install Virtualenv
 ------------------
-I install it with the package system of my OS, so that it ends in my
-global site-packages.
+To install globally with pip (if you have pip 1.3 or greater installed globally)::
 
-With Fedora/EL is just::
+  $ [sudo] pip install virtualenv
 
-  $ sudo yum install python-virtualenv
```

Comment:
From the sysadmin point of view, I think is better to use the package provided by the OS unless you need the capabilities of a newer version